hazelcast/src/main/java/com/hazelcast/quorum/QuorumCallback.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.quorum;
+
+/**
+ * QuorumCallback can be used to notify quorum service for a particular quorum result that originated externally.
+ */
+public interface QuorumCallback {
```

Comment:
It doesn't mean anything to me. If I hear a callback, I think E.g ExecutionCallback.. EventListener. All things that react on some change and can do stuff after that state change happened. 

But in this case I have the impression it is some kind of policy to control things.